misc/attach_disk_test.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+"""
+Test how Hail Batch attaches disks to instances. Motivation: job.storage(...) option
```

Comment:
FWIW I don't believe any of the 3 scripts in `misc` have been used since the pipeline started being used in production, and they're probably not worth migrating

Reply:
Good to know. I can mark them as not to migrate.